src/cluster/Cargo.toml

```diff
@@ -1,29 +1,32 @@
 [package]
 name = "fluvio-cluster"
-version = "0.3.1"
+version = "0.3.2"
 edition = "2018"
 license = "Apache-2.0"
 authors = ["Fluvio Contributors <team@fluvio.io>"]
 repository = "https://github.com/infinyon/fluvio"
 description = "Tools for installing and managing Fluvio clusters"
 
-
 [lib]
 name = "fluvio_cluster"
 path = "src/lib.rs"
 
+[[bin]]
+name = "fluvio-cluster"
+path = "src/main.rs"
+
 [features]
-default = ["native_tls"]
+default = ["cli", "native_tls"]
```

Comment:
cli is only needed for binary.  you can add `required-features` to [bin](https://stackoverflow.com/questions/35711044/how-can-i-specify-binary-only-dependencies)